extensions/amp-timeago/0.2/amp-timeago.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {PreactBaseElement} from '../../../src/preact/base-element';
+import {Timeago} from './timeago';
+import {isExperimentOn} from '../../../src/experiments';
+import {isLayoutSizeDefined} from '../../../src/layout';
+import {userAssert} from '../../../src/log';
+
+/** @const {string} */
+const TAG = 'amp-timeago';
+
+class AmpTimeago extends PreactBaseElement {
```

Comment:
Is the class name externally visible?

If so I would rather lean towards keeping it consistent with AMP naming convention so that if people are using something like NextJS to create Bento and AMP first pages they can convert between the two easily.

Reply:
It's not externally visible, this will only be used by AMP code in AMP mode.

Reply:
IMHO this should follow closely `amp-timeago`. We didn't make it `amp-time-ago`, which could have made sense earlier, but it's better to be consistent at this time with the element naming. I.e. if we decided to lint this name, our only way to do it would be using hyphen -> camelcase.